api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/TestConstants.java

```diff
@@ -15,4 +17,7 @@
     public static final int TOTAL_VOUCHER_ENTITLEMENT = 4;
     public static final int TOTAL_VOUCHER_VALUE_IN_PENCE = 1240;
 
+    public static final LocalDate DATE_OF_BIRTH = LocalDate.parse("1985-12-31");
```

Comment:
Should we prefix these with JAMES_ to indicate that they're all the same user?

Reply:
Done.